src/auro-button.js

```diff
@@ -247,3 +265,7 @@ export class AuroButton extends LitElement {
     `;
   }
 }
+
+// This is for compatibility with the old version of static properties
+// I REALLY want to use `static formAssociated = true;` but eslint doesn't like it
+AuroButton.formAssociated = true;
```

Comment:
have you tried adding static getter on line 87?

Reply:
fixed!